test/fixtures/transaction.json

```diff
@@ -229,60 +229,18 @@
         ]
       },
       "coinbase": false
-    }
-  ],
-  "hashForSignature": [
-    {
-      "description": "Out of range inIndex",
-      "txHex": "010000000200000000000000000000000000000000000000000000000000000000000000000000000000ffffffff00000000000000000000000000000000000000000000000000000000000000000000000000ffffffff01e8030000000000000000000000",
-      "inIndex": 2,
-      "script": "OP_0",
-      "type": 0,
-      "hash": "0000000000000000000000000000000000000000000000000000000000000001"
-    },
-    {
-      "description": "inIndex > nOutputs (SIGHASH_SINGLE)",
-      "txHex": "010000000200000000000000000000000000000000000000000000000000000000000000000000000000ffffffff00000000000000000000000000000000000000000000000000000000000000000000000000ffffffff01e8030000000000000000000000",
-      "inIndex": 2,
-      "script": "OP_0",
-      "type": 3,
-      "hash": "0000000000000000000000000000000000000000000000000000000000000001"
     },
     {
-      "txHex": "010000000200000000000000000000000000000000000000000000000000000000000000000000000000ffffffff00000000000000000000000000000000000000000000000000000000000000000000000000ffffffff01e8030000000000000000000000",
-      "inIndex": 0,
-      "script": "OP_0 OP_3",
-      "type": 0,
-      "hash": "3d56a632462b9fc9b89eeddcad7dbe476297f34aff7e5f9320e2a99fb5e97136"
-    },
-    {
-      "txHex": "010000000200000000000000000000000000000000000000000000000000000000000000000000000000ffffffff00000000000000000000000000000000000000000000000000000000000000000000000000ffffffff01e8030000000000000000000000",
-      "inIndex": 0,
-      "script": "OP_0 OP_CODESEPARATOR OP_3",
-      "type": 0,
-      "hash": "3d56a632462b9fc9b89eeddcad7dbe476297f34aff7e5f9320e2a99fb5e97136"
-    },
-    {
-      "txHex": "010000000200000000000000000000000000000000000000000000000000000000000000000000000000ffffffff00000000000000000000000000000000000000000000000000000000000000000000000000ffffffff01e8030000000000000000000000",
-      "inIndex": 0,
-      "script": "OP_0 OP_CODESEPARATOR OP_4",
-      "type": 0,
-      "hash": "fa075877cb54916236806a6562e4a8cdad48adf1268e73d72d1f9fdd867df463"
-    }
-  ],
-  "witness": [
-    {
       "description": "P2PK",
       "id": "8d4733995be9b3331a65eeb31bce24b213d4fabcbadf7f6634a8fea442c13e6a",
       "hash": "6a3ec142a4fea834667fdfbabcfad413b224ce1bb3ee651a33b3e95b9933478d",
       "hex": "010000000193aef40ae141694895e99e18e49d0181b086dd7c011c0241175c6eaf320099970000000049483045022100e57eba5380dcc8a7bdb5370b423dadd43070e1ca268f94bc97b2ded55ca45e9502206a43151c8af03a00f0ac86526d07981e303fc0daea8c6ed435abe8961533046d01ffffffff0160ea0000000000001976a914851a33a5ef0d4279bd5854949174e2c65b1d450088ac00000000",
-      "hasWitness": false,
-      "witnessHex": "",
+      "whex": "",
       "raw": {
         "version": 1,
         "ins": [
           {
-            "hash": "97990032af6e5c1741021c017cdd86b081019de4189ee995486941e10af4ae93",
```

Comment:
Bitcoin core has ambiguity between `hash` and an `id`.
For testing purposes,  we always call the `hash` the raw hex string which would have been returned by `sha256`, and the `id` is reversed.

This amends the test fixtures to reflect that.